parmed/gromacs/gromacstop.py

```diff
@@ -10,6 +10,10 @@
 from datetime import datetime
 import math
 import os
+try:
+    import pwd
+except ImportError:
+    pwd = None
```

Comment:
if not, `python setup.py install` will fail with ImportError for `pwd`. (it's dummy fix. lol).